public/components/metrics/redux/slices/metrics_slice.ts

```diff
@@ -111,18 +123,49 @@ const fetchRemoteMetrics = async (remoteDataSources: string[]): Promise<any> =>
   );
 };
 
+export const fetchOpenTelemetryIndices = async () => {
+  const { http } = coreRefs;
```

Comment:
We want to transition to `getOSDHttp` now -- it improves testability greatly.

Reply:
changed it